scripts/zones/Throne_Room/mobs/Shadow_Lord.lua

```diff
@@ -71,8 +71,11 @@ function onMobFight(mob,target)
         end
     else
         -- second phase AI: Implode every 9 seconds
-        if (mob:getBattleTime() % 9 == 0) then
```

Comment:
This was evaluated twice per second.  Thus implode twice every 9s, instead of once.